src/app/home/partners/partners.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="container center-align">
+  <p class="">Our Partners</p>
+  <div class="row">
+    <div class="card-image col s12 m3" *ngFor="let partner of partners | async">
+      <a href="//{{partner.link}}">
+        <img class="partner-logo" [src]="partner.logo_link">
```

Comment:
Add alt attribute to the image, you could use the partner name.

Reply:
Also, for firebase documents. for the attributes, use camel case. You have to map this to a model, and typescript uses camel case